hack/jenkins/upload-to-gcs.sh

```diff
@@ -63,21 +63,27 @@ readonly gcs_acl="public-read"
 readonly results_url=${gcs_build_path//"gs:/"/"https://console.cloud.google.com/storage/browser"}
 readonly timestamp=$(date +%s)
 
-function upload_version() {
-  echo -n 'Run starting at '; date -d "@${timestamp}"
-
-  # Try to discover the kubernetes version.
-  local version=""
+#########################################################################
+# Try to discover the kubernetes version.
+# prints version
+function find_version() {
   if [[ -e "version" ]]; then
-    version=$(cat "version")
+    cat version
   elif [[ -e "hack/lib/version.sh" ]]; then
-    version=$(
-      export KUBE_ROOT="."
```

Comment:
The indentation there was for the setting of `version`, but now the commands just run and output the result.  I only left the subshell there () so that KUBE_ROOT doesn't get reset in the calling scope.